ledger/src/blockstore_processor.rs

```diff
@@ -1374,13 +1375,15 @@ fn load_frozen_forks(
     cache_block_meta_sender: Option<&CacheBlockMetaSender>,
     timing: &mut ExecuteTimings,
     accounts_background_request_sender: &AbsRequestSender,
-) -> result::Result<u64, BlockstoreProcessorError> {
+) -> result::Result<(u64, usize), BlockstoreProcessorError> {
```

Comment:
Not a huge fan of returning tuple's of similar (u64/usize) types without some sort of documentation on what each is.

Could you update the comment for this function, preferably as a doc-comment?

😆 I see it's quite out of date anyway since this is definitely not returning a vector...

Reply:
Haha yup - this function bugged me a decent bit last night so I also have https://github.com/solana-labs/solana/pull/31475 in the pipeline which is just a handful of general cleanup.

That PR updates the doc comment, including specifying what each value in the tuple represents. I can pull the doc comment update into this PR tho